specs/dotNet4.5.1_should_pass.ps1

```diff
@@ -3,6 +3,9 @@ Framework "4.5.1x86"
 task default -depends MsBuild
 
 task MsBuild {
-  $output = &msbuild /version 2>&1
-  Assert ($output -NotLike "12.0") '$output should contain 12.0'
+    if ( $IsOSX -OR $IsLinux ) {}
+    else {
+        $output = &msbuild /version 2>&1
```

Comment:
Can you do me a favor and while you're changing these tests for the MSBuild version, can you add the `/nologo` parameter.  This will change the command output from multiline:
```
Microsoft (R) Build Engine version 14.0.25420.1
Copyright (C) Microsoft Corporation. All rights reserved.

14.0.25420.1
```
to a single line:
```
14.0.25420.1
```
There is another PR that touches these lines but I'm inclined not to accept it because it uses `Get-Command` to extract the MSBuild version number. I do not think that is portable to Linux/macOS.  So I think we're stuck with running MSBuild /version /nologo to get its version number.